src/js/components/deployments/__snapshots__/pastdeployments.test.js.snap

```diff
@@ -158,6 +158,95 @@ exports[`PastDeployments Component renders correctly 1`] = `
         </div>
       </div>
     </div>
+    <div
+      aria-expanded="false"
+      class="MuiAutocomplete-root MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon"
+      role="combobox"
+    >
+      <div
+        class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
+        style="margin-top: 0px;"
+      >
+        <label
+          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
+          data-shrink="false"
+          for="deployment-type-selection"
+          id="deployment-type-selection-label"
+        >
+          Filter by type
+        </label>
+        <div
+          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
+        >
+          <input
+            aria-autocomplete="list"
+            aria-invalid="false"
+            autocapitalize="none"
+            autocomplete="off"
+            class="MuiInputBase-input MuiInput-input MuiAutocomplete-input capitalized MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd"
```

Comment:
The "capitalized" class can be removed, it's forcing the placeholder text to be bit inconsistent

Reply:
👌 now it will only appear capitalized once there actually is text entered, so the placeholder will show up consistent with the rest